src/bin/compute.rs

```diff
@@ -7,8 +7,17 @@ use powersoftau::*;
 
 use std::fs::OpenOptions;
 use std::io::{self, Read, BufReader, Write, BufWriter};
+use std::path::Path;
+use std::process;
 
 fn main() {
+
+    // Check that the user placed the challenge file into the local directory
+    if !Path::new("./challenge").exists() {
+        println!("Error: ./challenge file not found.");
```

Comment:
panic! here would suffice instead of pulling in `std::process`.

Reply:
The idea here is that a regular error message is better for users, rather than a scary panic message.

If the compute binary is to be as bare bones as possible, where fewer dependencies is deemed better, I can change to panic.